diff --git a/data_generators.py b/data_generators.py
index 3b815de..06d1e05 100644
--- a/data_generators.py
+++ b/data_generators.py
@@ -13,11 +13,11 @@ import dgl
 
 from mazelib import Maze
 from mazelib.generate.Prims import Prims
-from gym_minigrid.envs.multiroom_mod import MultiRoomEnv
+from .gym_minigrid.envs.multiroom_mod import MultiRoomEnv
 
-from util import graph_metrics
-import util.transforms as tr
-import util.util as util
+from .util import graph_metrics
+from .util import transforms as tr
+from .util import util as util
 
 logger = logging.getLogger(__name__)
 
diff --git a/models/graphVAE.py b/models/graphVAE.py
index e461bba..10b695f 100644
--- a/models/graphVAE.py
+++ b/models/graphVAE.py
@@ -9,13 +9,12 @@ from torch import nn
 import torch.nn.functional as F
 from typing import Iterable
 
-from models.gnn_networks import GIN
-from models.networks import FC_ReLU_Network
-from util.distributions import sample_gaussian_with_reparametrisation, compute_kld_with_standard_gaussian, \
+from .gnn_networks import GIN
+from .networks import FC_ReLU_Network
+from ..util.distributions import sample_gaussian_with_reparametrisation, compute_kld_with_standard_gaussian, \
     evaluate_logprob_bernoulli, evaluate_logprob_one_hot_categorical, evaluate_logprob_diagonal_gaussian
-import util.transforms as tr
-import util.graph_metrics as gm
-
+from ..util import transforms as tr
+from ..util import graph_metrics as gm
 
 def graphVAE_elbo_pathwise(X, *, encoder, decoder, num_samples, elbo_coeffs, predictor=None, labels=None, permutations=None):
     # TODO: permutations not functional at the moment, because not implemented for Fx
diff --git a/models/networks.py b/models/networks.py
index c33571f..0f03471 100644
--- a/models/networks.py
+++ b/models/networks.py
@@ -1,6 +1,6 @@
 from torch import nn, Tensor
 from typing import Iterable, Tuple, Union
-from models.layers import *
+from .layers import *
 from math import prod
 import numpy as np
 
diff --git a/util/transforms.py b/util/transforms.py
index 08c1038..5433ff1 100644
--- a/util/transforms.py
+++ b/util/transforms.py
@@ -11,7 +11,7 @@ import torch
 import torch.nn.functional as F
 import torch.nn as nn
 
-from gym_minigrid.minigrid import MiniGridEnv, WorldObj, OBJECT_TO_IDX as Minigrid_OBJECT_TO_IDX, \
+from ..gym_minigrid.minigrid import MiniGridEnv, WorldObj, OBJECT_TO_IDX as Minigrid_OBJECT_TO_IDX, \
     IDX_TO_OBJECT as Minigrid_IDX_TO_OBJECT, COLOR_TO_IDX as Minigrid_COLOR_TO_IDX
 
 logger = logging.getLogger(__name__)
diff --git a/util/util.py b/util/util.py
index 163d1e7..86f5c9e 100644
--- a/util/util.py
+++ b/util/util.py
@@ -11,8 +11,8 @@ from matplotlib.colors import LinearSegmentedColormap
 import argparse
 from typing import Tuple, List
 
-from data_loaders import WrappedDataLoader
-import util.transforms as tr
+from ..data_loaders import WrappedDataLoader
+from . import transforms as tr
 
 #WIP section
 
